src/resources/plsPlusAddress/formModule.js

```diff
@@ -0,0 +1,165 @@
+{
+  options: {
+    form: {
+      evalContext: {
+        defaultPlsPlusApiKey: "auto",
+        plsPlusApiHost: "https://www.address.test-services.qld.gov.au",
+        getPlsPlusApiKey: function (key) {
+          if (key === "auto") {
+            const host = location.host;
+            if (host.indexOf("forgov") !== -1) {
+              return "forgov";
+            } else if (host.indexOf("tmr") !== -1) {
+              return "tmr";
+            } else if (host.indexOf("covid19") !== -1) {
+              return "covid19";
+            } else if (host.indexOf("health") !== -1) {
+              return "health";
+            } else if (host.indexOf("resource") !== -1) {
+              return "resource";
+            } else if (location.href.indexOf("www.qld.gov.au/dev/") !== -1 || location.href.indexOf("oss-uat.clients.squiz.net/dev/") !== -1) {
+              // if it is used in demo page
+              return "qldonline";
+            } else if (host.indexOf("api.forms.platforms") !== -1 || host.indexOf("formio") !== -1) {
+              // if it is used in formio dev environment
+              return "formiotest";
+            }
+          }
+          const validKeys = ["forgov", "tmr", "covid19", "health", "resource", "qldonline", "formiotest", "auto"];
+          if (validKeys.indexOf(key) === -1) {
+            return "invalid";
+            console.warn("Your PlsPlus API key is not valid, please enter a valid API key in formIO > form settings > custom properties > add value with `{ plsPlusApiKey: YOUR_FRANCHISE_API_KEY}`")
+          }
+          if (key === "auto") {
+            console.warn("We can't find a valid PlsPlus API key based on the URL, please enter a valid API key in formIO > form settings > custom properties > add value with `{ plsPlusApiKey: YOUR_FRANCHISE_API_KEY}`")
```

Comment:
might need link to wiki page with screen shots as its very confusing where it is (eg section, simliar to these screenshots https://help.form.io/userguide/forms )